java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/jdbc/Statement.java

```diff
@@ -0,0 +1,314 @@
+package com._4paradigm.openmldb.jdbc;
+import com._4paradigm.openmldb.SQLRouter;
+import com._4paradigm.openmldb.Status;
+
+import java.sql.*;
+
+public class Statement implements java.sql.Statement {
+    private com._4paradigm.openmldb.ResultSet resultSet;
+    private SQLRouter sqlRouter;
+    private boolean closed;
+
+    public Statement(SQLRouter router) {
+        this.sqlRouter = router;
+        this.closed = false;
+    }
+
+    @Override
+    public boolean execute(String sql) throws SQLException {
+        Status status = new Status();
+        if (resultSet != null) {
+            resultSet.delete();
+        }
+        resultSet = sqlRouter.HandleSQL(sql, status);
+        if (!status.IsOK()) {
+            String msg = status.getMsg();
+            status.delete();
+            throw new SQLException("executeSQL fail: " + msg);
+        }
+        status.delete();
+        if (resultSet != null) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public ResultSet getResultSet() throws SQLException {
+        if (resultSet == null) {
+            throw new SQLException("no result set");
+        }
+        return new SQLResultSet(resultSet);
+    }
+
+    @Override
+    public SQLResultSet executeQuery(String sql) throws SQLException {
+        Status status = new Status();
+        resultSet = sqlRouter.ExecuteSQL("", sql, status);
+        if (!status.IsOK()) {
+            String msg = status.getMsg();
+            status.delete();
+            throw new SQLException("executeSQL fail: " + msg);
+        }
+        status.delete();
+        return new SQLResultSet(resultSet);
+    }
+
+    @Override
+    public int executeUpdate(String sql) throws SQLException {
+        Status status = new Status();
+        com._4paradigm.openmldb.ResultSet rs = sqlRouter.HandleSQL(sql, status);
+        if (!status.IsOK()) {
+            String msg = status.getMsg();
+            status.delete();
+            throw new SQLException("executeSQL fail: " + msg);
+        }
+        status.delete();
+        if (rs != null) {
+            rs.delete();
+        }
+        return 0;
```

Comment:
Maybe return void if the return code is not used.

Reply:
function return type is `int`,  we cannot return null. And what the mean of `return void`?

Reply:
It's a defination of `java.sql.Statement.executeUpdate`, just return rowcount.

Reply:
> either (1) the row count for SQL Data Manipulation Language (DML) statements or (2) 0 for SQL statements that return nothing

`ExecuteInsert` method not support return row counts now. I post an issue and will implement later